modules/Bio/EnsEMBL/Compara/RunnableDB/Synteny/FetchSyntenyParametersAlignments.pm

```diff
@@ -144,6 +144,9 @@ sub _check_pairwise {
         or die sprintf("Could not find the method '%s' in the master database (%s)\n", $self->param('synteny_method_link_type'), $self->param('master_db'));
     my $master_synt_mlss = $self->param('master_dba')->get_MethodLinkSpeciesSetAdaptor->fetch_by_method_link_id_species_set_id($synt_method->dbID, $mlss->species_set->dbID);
 
+    # If 'from_first_release' has not been passed, the value of $release does
```

Comment:
I actually find this comment more confusing than helpful - suggest removing/moving/rewriting it.